src/ray/rpc/grpc_server.h

```diff
@@ -90,6 +94,8 @@ class GrpcServer {
   /// gRPC server can accept.
   std::vector<std::pair<std::unique_ptr<ServerCallFactory>, int>>
       server_call_factories_and_concurrencies_;
+  /// The factory for server side stream call.
+  std::vector<std::unique_ptr<ServerCallFactory>> server_stream_call_factories_;
```

Comment:
Could we put all these factories into one vector `server_call_factories_and_concurrencies_`, instead of creating another vector for stream call?